Rakefile

```diff
@@ -26,11 +26,23 @@ class TestSignableRequest < MAuth::Request
   def x_mws_authentication
     headers['X-MWS-Authentication']
   end
+
+  def mcc_time
+    headers['MCC-Time']
+  end
+
+  def mcc_authentication
+    headers['MCC-Authentication']
+  end
 end
 
 desc 'Runs benchmarks for the library.'
 task :benchmark do
-  mc = MAuth::Client.new(private_key: OpenSSL::PKey::RSA.generate(2048), app_uuid: SecureRandom.uuid)
+  mc = MAuth::Client.new(
+    private_key: OpenSSL::PKey::RSA.generate(2048),
+    app_uuid: SecureRandom.uuid,
+    v2_only_sign_requests: false
+  )
   authenticating_mc = MAuth::Client.new(mauth_baseurl: 'http://whatever', mauth_api_version: 'v1')
 
   stubs = Faraday::Adapter::Test::Stubs.new
```

Comment:
Sorry, not you, but can you fix: `DEPRECATION: Using `stub` from rspec-mocks' old `:should` syntax without explicitly enabling the syntax is deprecated. Use the new `:expect` syntax or explicitly enable `:should` instead. Called from /Users/ejennings/dev/mauth-client-ruby/Rakefile:53:in `block in <top (required)>'.`